emails/utils.py

```diff
@@ -174,14 +174,21 @@ def ses_send_raw_email(
     return HttpResponse("Sent email to final recipient.", status=200)
 
 
-def _start_message_with_headers(subject, from_address, to_address, reply_address):
+def _start_message_with_headers(
+    subject, from_address, to_address, reply_address, relay_address
+):
     # Create a multipart/mixed parent container.
     msg = MIMEMultipart("mixed")
     # Add subject, from and to lines.
     msg["Subject"] = subject
     msg["From"] = from_address
     msg["To"] = to_address
     msg["Reply-To"] = reply_address
+    if type(relay_address) == RelayAddress:
+        msg["X-RelayAddress-ID"] = str(relay_address.id)
```

Comment:
I'd use a namespace so we can add other relay-specific information
```suggestion
        msg["X-FirefoxRelay-RelayAddressId"] = str(relay_address.id)
```